### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by BucurEva87**
- created books and categories reducers
- configured redux store

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: { books: [] },
+  reducers: {
+    addBook: (state, book) => {
+      state.push(book);
+    },
+    removeBook: (state, bookIndex) => {
+      state.splice(bookIndex, 1);
```
Comment (EricMbouwe) [inline src/redux/books/books.js]: - [ ] You should define ADD_BOOK and REMOVE_BOOK as action types following the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: { categories: [] },
+  reducers: {
+    check: () => 'Under construction',
+  },
```
Comment (EricMbouwe) [inline src/redux/categories/categories.js]: - [ ]  You should define CHECK_STATUS as action type following the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @BucurEva87, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Use redux toolkit ✔
- Components ✔
- Linters ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (BucurEva87) [reply src/redux/books/books.js]: @EricMbouwe, I thought the whole idea of Redux Toolkit was, among other things, to relieve us of using those constant action types - given the fact that it's returning us actions based on the keys properties of the reducer property of the slice. I also got the idea that switches were no longer needed, if we used the Redux toolkit. If that's still necessary, however, of course, I will implement it.

Comment (EricMbouwe) [reply src/redux/books/books.js]: Hi @BucurEva87,

We still need action creators even with redux toolkit. The purpose of that is to follow the the core pattern action => reducer => store => action. You can learn more in this [section](https://redux-toolkit.js.org/usage/usage-guide) in the documentation

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @BucurEva87    🔥 

Good job on implementing the requested changes. You smashed it. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**